crates/fluvio-connector-deployer/src/local.rs

```diff
@@ -1,13 +1,52 @@
-use std::{
-    process::{Command, Stdio},
-    path::Path,
-    fs::canonicalize,
-};
-
-use anyhow::{Result, Context};
-use tracing::debug;
+use std::ffi::OsString;
+use std::fmt::{self, Display};
+use std::fs::canonicalize;
+use std::path::Path;
+use std::process::{Command, Stdio};
+
+use anyhow::{Context, Result};
+use tracing::{debug, warn};
+
 use crate::Deployment;
 
+#[derive(Copy, Clone, Debug, Default, PartialEq)]
+pub enum LogLevel {
+    Error,
+    Warn,
+    #[default]
+    Info,
+    Debug,
+    Trace,
+}
+
+impl Display for LogLevel {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self {
+            LogLevel::Error => write!(f, "error"),
+            LogLevel::Warn => write!(f, "warn"),
+            LogLevel::Info => write!(f, "info"),
+            LogLevel::Debug => write!(f, "debug"),
+            LogLevel::Trace => write!(f, "trace"),
+        }
+    }
+}
+
+impl From<OsString> for LogLevel {
```

Comment:
I guess this impl is not needed

Reply:
I think that it is needed for usage with `clap`